src/components/chat.js

```diff
@@ -310,12 +312,13 @@ class Chat extends Emitter {
             // publish the event and data over the configured channel
 
             // ensure the event exists within the global space
```

Comment:
Is the above comment stale now? Seems to be.

Reply:
yep